src/styles/blocks/card.css

```diff
@@ -0,0 +1,92 @@
+html {
+  font-family: Roboto, sans-serif;
+}
+
+.card {
+  width: 198px;
+  background-color: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding-top: 32px;
+}
+
+.card__content {
+  margin-left: 16px;
+  margin-right: 16px;
+}
+
+.card__img {
+  width: 160px;
+  height: 134px;
+  display: flex;
+  margin: 0 auto 40px;
+}
+
+.card__title {
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  color: #060b35;
+  margin-bottom: 4px;
+}
+
+.card_product-code {
+  font-size: 10px;
+  line-height: 14px;
+  color: #616070;
+  margin-bottom: 16px;
+}
+
+.card__reviews {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-bottom: 24px;
+}
+
+.card__reviews-total {
+  font-size: 10px;
+  color: #060b35;
+}
+
+.card__price {
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: 16px;
+}
+
+.card__price-text {
+  font-size: 12px;
+  line-height: 18px;
+  color: #616070;
+}
+
+.card__price-value {
+  font-weight: 700;
+  font-size: 16px;
+  line-height: 18px;
+  color: #060b35;
+}
+
+.card__buy {
+  display: flex;
+  justify-content: center;
+  font-weight: 700;
+  font-size: 14px;
+  line-height: 16px;
+  text-decoration: none;
+  text-transform: uppercase;
+  background: #00acdc;
+  border: 1px solid transparent;
+  border-radius: 5px;
+  color: #fff;
+  transition: 0.4s;
+  padding: 12px 0;
+  margin-bottom: 16px;
+}
```

Comment:
Its better to adds some spases betwen different rules groups like texe rules, position rules etc.